src/main/java/com/thehrdivision/UserEntity.java

```diff
@@ -0,0 +1,36 @@
+package com.thehrdivision;
+
+import com.sun.istack.NotNull;
+
+import javax.persistence.*;
+import java.util.Date;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+public class UserEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @NotNull
+    private String login;
+    @NotNull
+    private String password;
+    private String workEmail;
+    @OneToOne()
+    @JoinColumn(name = "position_id")
+    private Position position;
+    private Date createdAt;
```

Comment:
Tutaj wydaje mi się, że chcemy aby to pole zostało "zasilone" podczas insertu rekordu do bazy danych, tutaj więcej info -> 
https://stackoverflow.com/questions/221611/creation-timestamp-and-last-update-timestamp-with-hibernate-and-mysql/221827#221827

https://www.javafixing.com/2021/11/fixed-createddate-annotation-does-not.html

Co do typu Date, to tu któtki artykuł -> https://programminghints.com/2017/05/still-using-java-util-date-dont/